src/webots/nodes/utils/WbNodeOperations.cpp

```diff
@@ -26,6 +26,7 @@
 #include "WbParser.hpp"
 #include "WbProject.hpp"
 #include "WbProtoManager.hpp"
+#include "WbProtoModel.hpp"
```

Comment:
Not needed.
```suggestion
```

Reply:
Yes, this is needed.
`WbProtoModel` class is used at line 468 and 475.